tests/basic/test_service-cli_intgemm_8bit.bytearrayload.cpu-threads.4.sh

```diff
@@ -33,7 +32,7 @@ ARGS=(
     --ssplit-mode paragraph
     --beam-size 1
     --skip-cost
-    --shortlist $BRT_MODELS/deen/ende.student.tiny11/lex.s2t 50 50
+    --shortlist $BRT_MODELS/deen/ende.student.tiny11/lex.s2t.bin 50 50
```

Comment:
Does the Makefile fetch this file as of now? Some automagic in the archive?

Reply:
I moved this to `download-models.sh` (the gunzip and extra code you added for WASM something). Can you verify that bit is covered on a clean make install?

Reply:
@qianqianzhu updated the models, so if you redownload them it will work